src/lib/sidebar/tabs/Guides.svelte

```diff
@@ -81,6 +84,11 @@
   <SidebarPage label="Array Iteration" icon={IconList} page="/guide/nbt-and-scores/array-iteration" />
 </SidebarCategory>
 
+<SidebarCategory name="Performance" icon={IconBrandSpeedtest}>
+  <SidebarPage label="How to measure" icon={IconRuler2} page="/guide/performance/how-to-measure" />
+  <SidebarPage label="Write optimized code" icon={IconMathMaxMin} page="/guide/performance/write-optimized-code" />
```

Comment:
```suggestion
  <SidebarPage label="How to Measure Performance" icon={IconRuler2} page="/guide/performance/how-to-measure" />
  <SidebarPage label="Writing Optimized Code" icon={IconMathMaxMin} page="/guide/performance/write-optimized-code" />
```

I would prefer titles be a form of title case, but the other title clarifications are optional